src/styles/blocks/_header.scss

```
@@ -0,0 +1,19 @@
+.header {
+  &__wr {
+    display: flex;
+    padding: 0 50px;
+    justify-content: space-between;
+    align-items: center;
+    max-width: 100%;
+    box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.05);
+
+    &__logo {
+      align-items: center;
+  justify-content: center;
+  height: 40px;
+  width: 40px;
+  display: block;
+    }
+  }
+}
```

header__logo should be nested in header not header__wr
```suggestion
.header {
  &__wr {
    display: flex;
    padding: 0 50px;
    justify-content: space-between;
    align-items: center;
    max-width: 100%;
    box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.05);
  }
  
  &__logo {
    align-items: center;
    justify-content: center;
    height: 40px;
    width: 40px;
    display: block;
  }
}
```